_includes/header.html

```diff
@@ -72,7 +72,7 @@ <h1 class="Header-titleLink">
       </div>
       {% if include.hasSearch %}
       <div class="Header-search ml-4 mb-0" id="header-search">
-        {% include search_form.html %}
+        <search></search>
```

Comment:
Custom elements should have a dash in their name.

Reply:
Do you mean like `custom_search`?

Reply:
Something like `<custom-search>`

Reply:
yes, my brain thought `custom-search` but my hands wrote `custom_search` 😅

Reply:
I've added the `custom` prefix in this [commit](https://github.com/italia/developers.italia.it/commit/99356c7fdd3e9c59dd1b0cd2d8328625aa47ae84)